pom.xml

```diff
@@ -103,12 +84,21 @@
         </dependency>
 
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>1.7.30</version>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.1</version>
             <scope>test</scope>
         </dependency>
 
+
+        <dependency>
+            <groupId>org.mortbay.jetty</groupId>
+            <artifactId>jetty-servlet-tester</artifactId>
+            <version>6.1.16</version>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
what is the purpose for that?

Reply:
That needs for testing my servlet. If you look in my "TestingWebApplicationTest" i created "ServletTester" and "HttpTester"

Reply:
This is redundant. Use TestRestTemplate. Keep on reading about SpringBoot testing

Reply:
I remowed that dependency and rewrite test when i used TestRestTemplate. I continue read about springBoot testing